cmd/kube-controller-manager/app/core.go

```diff
@@ -143,14 +145,33 @@ func startNodeIpamController(ctx ControllerContext) (http.Handler, bool, error)
 		}
 	}
 
+	nodeCIDRMaskSizes, err := processNodeCIDRMasks(ctx.ComponentConfig.NodeIPAMController.NodeCIDRMaskSize)
+	if err != nil {
+		return nil, false, err
+	}
+	// failure: length of node cidr masks greater than length of cluster cidrs
+	if len(nodeCIDRMaskSizes) > len(clusterCIDRs) {
+		return nil, false, fmt.Errorf("len of node cidr masks is %v > len of cluster cidrs %v", len(nodeCIDRMaskSizes), len(clusterCIDRs))
```

Comment:
I think `len` is a bit cryptic when searching through the logs, I think something like this is clearer:
```
return nil, false, fmt.Errorf("# of node CIDR masks is %v which is greater than the # of cluster CIDRs", len(nodeCIDRMaskSizes), len(clusterCIDRs))
```

Reply:
the symbol # is very specific for the english/american language (don't know exactly :sweat_smile: ) , can we use `length` or `size` instead?

Reply:
Not sure about `size` and `length` because that implies the actual size of the cidr, thoughts on `number`?

Reply:
"number" or "num" please